tests/ext/test_sentry.py

```diff
@@ -3,29 +3,48 @@
 from loafer.ext.sentry import sentry_handler
 
 
+class MockScope:
+    def __init__(self):
+        self.set_extra = mock.Mock()
+
+    def __enter__(self):
+        return self
+
+    def __exit__(self, exc_type, exc_value, traceback):
```

Comment:
:lipstick:  Rename the class to "DummyScope" (it serves to mock, but it's not a mock subclass)  and use `*args` here (since we are not interested in the method signature).

Reply:
I'm thinking about replacing this class with a `MagicMock`.

It would become something like:
```python
    mock_scope = mock.MagicMock()
    sdk_mocked = mock.Mock()
    sdk_mocked.push_scope.return_value = mock_scope
    ...
    mock_scope.__enter__.return_value.set_extra.assert_called_once_with(
        "message", "test"
    )
```

What do you think? :thinking:

Reply:
Does it provide `__enter__` method? If so, that would be great.

Reply:
Another thing, pls add yourself to `CONTRIBUTORS.rst `.

Reply:
> Does it provide `__enter__` method? If so, that would be great.

As far as I know, `MagicMock` should be used when you need those magic methods. I will give it a try.